### ISSUE TITLE
error: can't read default.nnw

### ISSUE THREAD

**Issue by sndik16** [astromatic/sextractor#49]
Hello, I get an error when running the program "ERROR: can't read default.nnw".
I found on internet somewhere advice to transfer the default.nnw from config directory in the program, to my working directory with images.

I did that, along with default.conv, default.sex, default.psf (from tests) and default.param. But is this correct? The results I get seem reasonable.

Is it correct that I need to put those files into my working directory?

Thank you,
Maria

**Comment by kirxkirx** [comment]
Hi Maria,

In short - yes, this is correct. 

- default.sex - is the file that controls SExtractor (things like detection threshold, background modelling details). You will likely need to fine-tune it for your data.
- default.conv - is the convolution kernel applied to the input image before running source detection. The default kernel works fine in the great majority of cases. The only cases where I saw a noticeable improvement from using a custom-tailored kernel were images with _really_ small stars, like the image obtained with a DSLR camera having ~100 mm focal length lens... or TESS).
- default.psf - seems to be an output file of SExtractor's sister program PSFEx. It's a PSF model that SExtractor can use to do PSF-fitting photometry. It is not needed unless you specifically do PSF-fitting photometry.
- default.param - controls SExtractor output. You'll likely need to modify it to suit your needs.
- default.nnw - neural network weights for star/galaxy separation. Personally, I didn't find it very useful in practice.

In summary: files default.sex, default.param, default.conv and default.nnw should be in your working directory and you probably want to edit the first two files with a text editor to fine-tune SExtractor for your images and use case.
